cloudify_cli/tests/commands/test_deployments.py

```diff
@@ -950,3 +952,305 @@ def test_deployment_modifications_get(self):
+        self.assertRaises(
+            ClickInvocationException,
+            self.invoke,
+            'cfy deployments schedule create backup -d dep1'
+        )
+
+    def test_deployment_schedule_create_missing_deployment_id(self):
+        self.assertRaises(
+            ClickInvocationException,
+            self.invoke,
+            'cfy deployments schedule create uninstall -s "12:33"'
+        )
+
+    def test_deployment_schedule_create_bad_time_expressions(self):
+        self.client.execution_schedules.create = MagicMock(
+            return_value=execution_schedules.ExecutionSchedule({}))
+        command = 'cfy deployments schedule create install -d dep1 -s {}'
+        error_msg = '{} is not a legal time format. accepted formats are ' \
+                    'YYYY-MM-DD HH:MM | HH:MM'
+
+        self.invoke(command.format('blah'),
+                    err_str_segment=error_msg.format('blah'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"15:33:18"'),
+                    err_str_segment=error_msg.format('15:33:18'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('99:99'),
+                    err_str_segment=error_msg.format('99:99'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"2000/1/1 09:17"'),
+                    err_str_segment=error_msg.format('2000/1/1 09:17'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"-1 min"'),
+                    err_str_segment=error_msg.format('-1 min'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"+10 dobosh"'),
+                    err_str_segment='10 dobosh is not a legal time delta',
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"+rez"'),
+                    err_str_segment='+rez is not a legal time delta',
+                    exception=CloudifyCliError)
+
+    def test_deployment_schedule_create_bad_timezone(self):
+        self.invoke('cfy deployments schedule create install -d dep1 -s '
+                    '"7:15" --tz Mars/SpaceX',
+                    err_str_segment='Mars/SpaceX is not a recognized timezone',
+                    exception=CloudifyCliError)
+
+    def test_deployment_schedule_create_months_delta(self):
+        self.client.execution_schedules.create = MagicMock(
+            return_value=execution_schedules.ExecutionSchedule({}))
+        self.invoke('cfy deployments schedule create backup -d dep -s "+13mo"')
+        call_args = list(self.client.execution_schedules.create.call_args)
+
+        current_month = datetime.datetime.utcnow().month
+        current_year = datetime.datetime.utcnow().year
+        expected_month = 1 if current_month == 12 else current_month+1
+        expected_year = current_year + (2 if current_month == 12 else 1)
+        expected_since = datetime.datetime.utcnow().replace(
```

Comment:
This comment holds to the whole PR.
How about adding an import of `from datetime import datetime`? 

And besides, I would prefer creating the variable `now = datetime.datetime.utcnow()` and just use it along the way.

Reply:
`from datetime import datetime` will compel me to change a lot of code lines which have nothing to do with this PR
addressed your other comment